src/Microsoft.PowerShell.MarkdownRender/VT100EscapeSequences.cs

```diff
@@ -0,0 +1,284 @@
+        /// </summary>
+        /// <param name="headerText">Text of the header to format.</param>
+        /// <returns>Formatted Header 6 string.</returns>
+        public string FormatHeader6(string headerText)
+        {
+            return string.Concat(Esc, options.Header6, headerText, endSequence);
+        }
+
+        /// <summary>
+        /// Class to represent default VT100 escape sequences.
+        /// </summary>
+        /// <param name="codeText">Text of the code block to format.</param>
+        /// <param name="isInline">True if it is a inline code block, false otherwise.</param>
+        /// <returns>Formatted code block string.</returns>
+        public string FormatCode(string codeText, bool isInline)
+        {
+            if (isInline)
+            {
+                return string.Concat(Esc, options.Code, codeText, endSequence);
+            }
+            else
+            {
+                // For code blocks, [500@ make sure that the whole line has background color.
+                return string.Concat(Esc, options.Code, codeText, Esc, "[500@", endSequence);
+            }
+        }
+
+        /// <summary>
+        /// Class to represent default VT100 escape sequences.
+        /// </summary>
+        /// <param name="linkText">Text of the link to format.</param>
+        /// <param name="url">URL of the link.</param>
+        /// <param name="hideUrl">True url should be hidden, false otherwise. Default is true.</param>
+        /// <returns>Formatted link string.</returns>
+        public string FormatLink(string linkText, string url, bool hideUrl = true)
+        {
+            if (hideUrl)
+            {
+                return string.Concat(Esc, options.Link, "\"", linkText, "\"", endSequence);
+            }
+            else
+            {
+                return string.Concat("\"", linkText, "\" (", Esc, options.Link, url, endSequence, ")");
+            }
+        }
+
+        /// <summary>
+        /// Class to represent default VT100 escape sequences.
+        /// </summary>
+        /// <param name="emphasisText">Text to format as emphasis.</param>
+        /// <param name="isBold">True if it is to be formatted as bold, false to format it as italics.</param>
+        /// <returns>Formatted emphasis string.</returns>
+        public string FormatEmphasis(string emphasisText, bool isBold)
+        {
+            var sequence = isBold ? options.EmphasisBold : options.EmphasisItalics;
+            return string.Concat(Esc, sequence, emphasisText, endSequence);
+        }
+
+        /// <summary>
+        /// Class to represent default VT100 escape sequences.
+        /// </summary>
+        /// <param name="altText">Text of the image to format.</param>
+        /// <returns>Formatted image string.</returns>
+        public string FormatImage(string altText)
+        {
+            return string.Concat(Esc, options.Image, "[", altText, "]", endSequence);
```

Comment:
nice, i wonder if you should check for empty string and include this as "image"

Reply:
fixed.